AEPMedia/Tests/UnitTests/MediaCollectionReportHelperTests.swift

```diff
@@ -22,7 +22,25 @@ class MediaCollectionReportHelperTests: XCTestCase {
         // Setup
         let host = "abc.com"
         // Action
-        let url = MediaCollectionReportHelper.getTrackingURL(host: host)
+        let url = MediaCollectionReportHelper.getTrackingURL(trackingServer: host)
+        // Assert
+        XCTAssertEqual(url, URL(string: "https://\(host)/api/v1/sessions"))
+    }
+
+    func testGetTrackingUrl_domainContainsValidPath() {
+        // Setup
+        let host = "abc.com/va/"
+        // Action
+        let url = MediaCollectionReportHelper.getTrackingURL(trackingServer: host)
+        // Assert
+        XCTAssertEqual(url, URL(string: "https://\(host)/api/v1/sessions"))
+    }
+
+    func testGetTrackingUrl_domainContainsInvalidPath() {
+        // Setup
+        let host = "//abc.com/va//"
+        // Action
+        let url = MediaCollectionReportHelper.getTrackingURL(trackingServer: host)
         // Assert
         XCTAssertEqual(url, URL(string: "https://\(host)/api/v1/sessions"))
```

Comment:
Should the expected result be `https://abc.com/va/api/v1/sessions` instead of `https:////abc.com/va///api/v1/sessions` ?

Reply:
As per our slack discussion: we usually don't do any validation on the tracking server value provided and let customer reconcile from the errors if they provide invalid url. Also launch has a suggested format to guide them. The above test case is just the side effect of our logic.